Terminal.Gui/Views/TextView.cs

```diff
@@ -392,14 +392,34 @@ void ColorSelection ()
 				Driver.SetAttribute (ColorScheme.Normal);
 		}
 
-		bool isReadOnly;
+		bool isReadOnly = false;
+
+		/// <summary>
+		/// Indicates readonly attribute of TextView
+		/// </summary>
+		/// <value>Boolean value(Default false)</value>
 		public bool ReadOnly {
 			get => isReadOnly;
 			set {
 				isReadOnly = value;
 			}
 		}
 
+		int nPageShift = 5;
+
+		/// <summary>
+		/// Number of lines to process for PageUp and PageDown keypress event
+		/// </summary>
+		/// <value>Number of lines(Default 5).</value>
+		public int PageShift 
+		{
+			get => nPageShift;
+			set 
+			{
+				nPageShift = value;
+			}
+		}
+
```

Comment:
We should likely change this to be computed and follow the same rules that Emacs uses based on the height of the view.

Reply:
Can you point me to an example in the code?